UM/Resources.py

```diff
@@ -264,7 +264,7 @@ def __find(cls, resource_type, *args):
     @classmethod
     def __initializeStoragePaths(cls):
         if platform.system() == "Windows":
-            cls.__config_storage_path = os.path.join(os.path.expanduser("~\\AppData\\Local\\"), cls.ApplicationIdentifier)
+            cls.__config_storage_path = os.path.join(os.getenv("LOCALAPPDATA"), cls.ApplicationIdentifier)
```

Comment:
In fact, I very consciously used a method in Arcus that is unsupported on XP. So yes, trying to run on XP will break. Which is why we do not support it.